Modules/Twp/Entities/IndigeneityType.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Modules\Twp\Entities;
+
+class IndigeneityType extends ModuleModel
+{
+
+    protected $fillable = ['title', 'is_active'];
+
+    public function students()
+    {
+        return $this->belongsToMany(
```

Comment:
This should be enough        return $this->belongsToMany('Modules\Twp\Entities\Student', 'indigeneity_student');